prime-router/src/main/kotlin/Element.kt

```diff
@@ -910,7 +910,9 @@ data class Element(
         // someone passes through character values, like a name, or some other text info. this is not
         // attempting to check and see if the number is properly formatted, just a short circuit to
         // fail out on REALLY bad data
-        private val maybeAPhoneNumber = Regex("""[\d\(\)\+x\- ]+""")
+        private val maybeAPhoneNumber = Regex(
+            """(\+\d{1,4}\s?-?)?\(?(\d{1,3})\)?\s?-?(\d{3,4})\s?-?(\d{3,4})\s?((x)?(ext.)?(#)?\s?\d+)?"""
```

Comment:
Should I put a comment explaining the regex?

Reply:
Good catch! We think a like, I was debating whether to use the OR operator, I think it makes it look cleaner.